src/build/optimize-streams.ts

```diff
@@ -132,7 +132,11 @@ export class JSDefaultMinifyTransform extends JSBabelTransform {
   constructor() {
     super({
       presets:
-          [babiliPreset(null, {'unsafe': {'simplifyComparisons': false}})]
+          [minifyPreset(null, {
+            flipComparisons: true,
+            guards: true,
+            typeConstructors: true
```

Comment:
@TimvdLippe are these settings equivalent to the simplifyComparisons setting or are there any meaningful differences we should note in CHANGELOG etc?

Reply:
Hm, maybe a plugin has to be included for that now: https://github.com/babel/minify/tree/master/packages/babel-plugin-transform-simplify-comparison-operators, but given that the plugin was not supposed to run, not including it has the same meaning? Not 100% on that.

Reply:
/derp I just noticed your description mentioning the paren-removal change identified by tests.

Reply:
It looks like `simplifyComparisons` was hoisted out of `unsafe` into top level of options.  I'm able to get the tests to pass with the following...

```typescript
   constructor() {
     super({
       presets:
          [minifyPreset(null, {simplifyComparisons: false})]
     });
   }
```

Was that missed or do these options matter in other ways not covered:
```
            flipComparisons: true,
            guards: true,
            typeConstructors: true
```

Reply:
Hm that is really weird, I couldn't find it in the options. If that fixes it, then yes please! Feel free to commit to this branch :smile:

Reply:
ok will push